pkg/console/aws.go

```diff
@@ -28,11 +28,39 @@ type awsSession struct {
 	SessionToken string `json:"sessionToken"`
 }
 
+func (a AWS) URLs(creds aws.Credentials) ([]string, error) {
+
+	urls := []string{}
+
+	//if region and service were not specified create a single default url and return
+	if a.Region == "" && a.Service == nil {
```

Comment:
this check will be safer
```suggestion
	if a.Region == "" && len(a.Service) == 0 {
```